shell/platform/tizen/flutter_tizen_engine_unittest.cc

```diff
@@ -65,49 +34,53 @@ class FlutterTizenEngineTest : public ::testing::Test {
     engine_ = nullptr;
   }
 
-  std::string assets_path_;
-  std::string icu_data_path_;
-  std::string aot_lib_path_;
   flutter::FlutterTizenEngine* engine_;
-  FlutterDesktopEngineProperties engine_prop_ = {};
-  std::vector<const char*> switches_;
+};
+
+class FlutterTizenEngineTestHeaded : public FlutterTizenEngineTest {
+ protected:
+  void SetUp() {
+    FlutterTizenEngineTest::SetUp();
+    engine_->InitializeRenderer(0, 0, 800, 600);
+  }
 };
 
 TEST_F(FlutterTizenEngineTest, Run) {
   EXPECT_TRUE(engine_ != nullptr);
-  EXPECT_TRUE(engine_->RunEngine(engine_prop_));
+  EXPECT_TRUE(engine_->RunEngine());
   EXPECT_TRUE(true);
```

Comment:
I guess we can remove this line. It is the result of my laziness. :smile:

Reply:
Oh, I thought you intentionally added the assert.